entity-service-impl/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
   implementation("com.github.f4b6a3:uuid-creator:3.5.0")
   implementation("io.reactivex.rxjava3:rxjava:3.0.11")
   implementation("com.google.guava:guava:30.1.1-jre")
+  implementation("org.apache.commons:commons-lang3:3.11")
```

Comment:
Not worth a second util lib just for pair.

Reply:
Will remove